src/cli.ts

```diff
@@ -162,6 +162,8 @@ function configIcon(
 
 	if (background) {
 		config.background = background;
+	} else {
+		config.background = 'white';
```

Comment:
Hi! Thank you for contributing! I think `transparent` will be better.

Reply:
Your readme says the default is `white` ;-)

Reply:
‑‑background, -b | Background color for icons and startup images. | white
-- | -- | --